e2e-polybft/consensus_test.go

```diff
@@ -126,41 +133,97 @@ func TestE2E_Consensus_RegisterValidator(t *testing.T) {
 			ValidatorSetAddr:  contracts.ValidatorSetContract},
 		&e2eStateProvider{txRelayer: txRelayer})
 
-	// create new account
-	addrs, err := cluster.InitSecrets(newValidatorSecrets, 1)
+	// create the first account and extract the address
+	addrs, err := cluster.InitSecrets(firstValidatorDataDir, 1)
 	require.NoError(t, err)
 
-	// extract new validator address
-	newValidatorAddr := ethgo.Address(addrs[0])
+	firstValidatorAddr := ethgo.Address(addrs[0])
+
+	// create the second account and extract the address
+	addrs, err = cluster.InitSecrets(secondValidatorDataDir, 1)
+	require.NoError(t, err)
 
-	// assert that account is created
+	secondValidatorAddr := ethgo.Address(addrs[0])
```

Comment:
Are we indexing correct address here?
```suggestion
	secondValidatorAddr := ethgo.Address(addrs[1])
```

Reply:
We do, the first and the second address are created separately

Reply:
Leftover comment, deleted it from VS Code, but it remained there by mistake :smile: